site/_posts/2023-03-10-growing-is-hard.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog-post
+title: Building a team is hard&#58; part I
+description: Building a team is harder than it looks. The attrition problem.
+author: Daniel DeGroff
+category: article
+image: blogs/consents-app/managing-consent-header.png
+tags: startup bootstrap dev hiring firing attrition
+excerpt_separator: "<!--more-->"
+---
+
+Building a engineering team is hard. There are many reasons for this, and one of them is attrition. In this article I'll talk about my pain and suffering relate to attrition while building FusionAuth. 
+
+<!--more-->
+
+### Statistics
+
+The average tenure of a software engineer varies. Interestingly it may even be the lowest in the largest tech companies such as Facebook (Meta), Amazon, Apple, Netflix and Google, commonly referred to as "FAANG".
+
+One report shows an average tenure in big tech is between 1.1 and 3.2 years.[^1] The US Bureau of Labor and Statistics report a nationwide average of 3.8 to 4.3 years which is not limited to the software industry.[^2] Regardless of which statistic you use, it is hard to argue that average tenure is short, and almost certainly lower than the average for the general US population.
+
+### Bye, Felicia
+Switching jobs means new experiences, more money and a change of scenery. This can be very appealing early in your career and can help you level up quickly.
+
+The downside is that you rarely develop the same level of experiences and skill that you would by sticking around, learning to work through conflict, mastering a code base and learning how to stay productive in spite of company politics.
+
+Unfortunately this doesn't seem to be the sentiment shared by many engineers. 
+
+### The problem
+By the time an employee is beginning to produce and become competent - they have one foot out the door looking for their next adventure. 
+
+There are various ways to overcome this problem.
+
+Venture capital backed companies solve this by hiring many engineers at once. Using this strategy, even if a few leave, there is nearly always overlap in tenure so that the combined skills on the team never go to zero.
```

Comment:
```suggestion
Venture Capital-backed companies solve this by hiring many engineers at once. Using this strategy, even if a few leave, there is nearly always overlap in tenure so that the combined skills on the team never go to zero.
```